### ISSUE TITLE
Tutorial 0 - Error: Unable to locate HelloWorld main module file

### ISSUE THREAD

**Issue by HelgeCPH** [pybee/briefcase#34]
Hej,

I am experimenting with Briefcase. I cannot get the Tutorial 0 (https://briefcase.readthedocs.io/en/latest/tutorials/tutorial-0.html#) to work on my machine. Likely it is something obvious, but I do not know what to look for.

After following the steps of the tutorial, XCode prints the following error into the console instead of the `Hello, World!` message:

```
2017-04-22 01:17:57.647 Hello World[74577:3342107] PythonHome is: /Users/thisisme/Library/Developer/CoreSimulator/Devices/BD73FC4D-D82B-4296-A6F6-B8A75806BFC4/data/Containers/Bundle/Application/9BBFB0DA-AF7E-4CB8-B48B-E382F34B7616/Hello World.app/Library/Python.framework/Resources
2017-04-22 01:17:57.649 Hello World[74577:3342107] PYTHONPATH is: PYTHONPATH=/Users/thisisme/Library/Developer/CoreSimulator/Devices/BD73FC4D-D82B-4296-A6F6-B8A75806BFC4/data/Containers/Bundle/Application/9BBFB0DA-AF7E-4CB8-B48B-E382F34B7616/Hello World.app/Library/Application Support/org.example.HelloWorld/app:/Users/thisisme/Library/Developer/CoreSimulator/Devices/BD73FC4D-D82B-4296-A6F6-B8A75806BFC4/data/Containers/Bundle/Application/9BBFB0DA-AF7E-4CB8-B48B-E382F34B7616/Hello World.app/Library/Application Support/org.example.HelloWorld/app_packages
2017-04-22 01:17:57.650 Hello World[74577:3342107] Initializing Python runtime
2017-04-22 01:17:57.787 Hello World[74577:3342107] Unable to locate HelloWorld main module file
```

I am OS X 10.11.6, XCode Version 8.2.1., `virtualenvwrapper` to manage my virtualenvs, and I am in an virtualenv with a Python in version 3.5.2.

I followed the steps tutorial:

```
mkdir tutorial
cd tutorial/
mkdir iostutorial
cd iostutorial/
mkdir HelloWorld
touch HelloWorld/__init__.py
echo 'print("Hello, World!")' > HelloWorld/app.py
touch setup.py
nano setup.py
```

Into the `setup.py`, I copied the code snippet given in the tutorial.
After the steps above, my project looks as in the following:

```
 └── tutorial
     └── iostutorial
         ├── HelloWorld
         │   ├── __init__.py
         │   └── app.py
         └── setup.py
```

Subsequently, I build the project, open it with XCode, and run it there (via the play button)

```
python setup.py ios
open iOS/Hello\ World.xcodeproj/
```

After the Simulator started up I get the error message from the top.

What am I doing wrong?  What do I have to fix?

**Comment by freakboy3742** [comment]
Looks like you got caught between a code change an a tutorial update. For compatibility with other platforms, we had to make a small change a couple of days ago to the way briefcase expects an app to start. Apps now need a `__main__.py` file in addition to `app.py` and `__init__.py`

Unfortunately, the tutorial was lagging behind. I've just pushed an update to the tutorial that addresses the problem. Apologies for the confusion!
